packages/editor/src/components/post-actions/actions.js

```diff
@@ -1007,6 +918,7 @@ export const duplicateTemplatePartAction = {
 };
 
 export function usePostActions( { postType, onActionPerformed, context } ) {
+	useDefaultActions();
```

Comment:
I think this is probably not the right place to use this hook. 

Because ideally, this only run ones and third-parties should be able to unregister the core actions even before they are actually rendered. So I'm not sure what the best path forward here:

 - Move this call to the top level component in both post and site editors? Not ideal to have to duplicate this call. 
 - Avoid using a hook and just call this top level, when loading the module. 
 - Use a "filter" instead of suggesting to third-parties to use the unregister functions?

@gziolo maybe you have better ideas here since you worked on similar things for blocks registrations..

Reply:
I think the "Move this call to the top level component in both post and site editors?" is the better approach. Could we put on some top-level editor component used by both edit site and edit post so we avoid the duplication?

Reply:
The site editor is not just an "Editor", it's the equivalent of the "whole admin". The editor can be unmounted there.

Reply:
Good point in that case I would still opt for "Move this call to the top level component in both post and site editors?" and would duplicate the call on edit post and edit site, it is just one line of code.